Documentation/fsck-msgids.txt

```diff
@@ -19,6 +19,9 @@
 `badParentSha1`::
 	(ERROR) A commit object has a bad parent sha1.
 
+`badRefName`::
+	(ERROR) A ref has a bad name.
```

Comment:
Nit: might be better to say `A ref has an invalid format.` similar to the one below.

Reply:
OK